bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -86,6 +87,17 @@ impl fmt::Display for PackedLockTime {
     }
 }
 
+impl FromHexStr for PackedLockTime {
+    type Error = core::num::ParseIntError;
+
+    fn from_hex_str_no_prefix<S: AsRef<str> + Into<String>>(s: S) -> Result<Self, FromHexError<Self::Error>>
+        where Self: Sized
```

Comment:
```suggestion
    where
        Self: Sized
```

Can you run the formatter once you are ready to merge please. We are adding a new file here and not excluding it from `rustfmt` in `rustfmt.toml` so `cargo +nightly fmt` will format it for you.

Reply:
Sure, I somehow broke my format on save and completely forgot it existed. Will make sure this is done in the future. Thanks

Reply:
Be aware that we use the nightly toolchain to format so if you have your editor configured with the stable toolchain then you'll get loads of unwanted changes on save.